phpcs.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <ruleset name="Project Rules">
-    <rule ref="Bluehost"/>
-    <config name="testVersion" value="5.3-"/>
-    <config name="minimum_supported_wp_version" value="4.7"/>
-</ruleset>
+    <rule ref="Newfold" />
+    <config name="testVersion" value="5.3-" />
+    <config name="minimum_supported_wp_version" value="4.7" />
```

Comment:
I'm not sure what your policy is for backward compatibility with WordPress, but our team typically only supports the 3 latest versions. It may be worth running PHPCS to see if you are using any functions that only exist in newer versions and then update the version here accordingly.